### ISSUE TITLE
Error in TpetraLinearSystemHelpers.C

### ISSUE THREAD

**Issue by marchdf** [Exawind/nalu-wind#407]
Using the master branch (hash: 572e0b5096ca786), I am getting the following error: 
```
IOSS: Using decomposition method 'RCB' for 344,064 elements on 72 processors.
terminate called after throwing an instance of 'std::logic_error'
  what():  Requirement( bulk.is_valid(master) ) FAILED
Error occured at: nalu-wind/src/TpetraLinearSystemHelpers.C:124
Error: get_entity_master, P0 failed to get entity for naluId=6, from entity with stkId=4, owned=0, shared=0,  elems:

srun: error: r1i0n13: task 0: Aborted (core dumped)
```

This setup used to run and it runs fine if I just use Hypre.

Attached is the input file and I will email the mesh (26MB)

[flatPlate.txt](https://github.com/Exawind/nalu-wind/files/3718125/flatPlate.txt)

**Comment by alanw0** [comment]
ok I'll look into this and figure out what's going on. Do you know how long ago it used to run?

**Comment by marchdf** [comment]
From timestamps on folders it looks like sometime around July 5-12 before I got sucked into TAMS stuff.

**Comment by sayerhs** [comment]
@alanw0 It seems like something related to  #338

**Comment by alanw0** [comment]
@sayerhs Yes it definitely is. I have a fix, running his case all the way through now. I'll get a pull-request in soon.

**Comment by sayerhs** [comment]
Fixed in #409
